src/main/drivers/mcu/apm32/bus_spi_apm32.c

```diff
@@ -0,0 +1,450 @@
+/*
```

Comment:
(@SteveCEvans)
There is a lot of MCU-independent logic in this file. IS it possible to separate from SPI impelmentations?